### ISSUE TITLE
"Text" of a text line directly above Go-to command is never visible

### ISSUE THREAD

**Issue by Wipeless** [ImpowerGames/impower#169]
**Describe the bug**

**Steps To Reproduce**
1. Create the following script:
```
# Root

Time to try
> {~*|First|Second|Third}

^

# First

First try

> Root

# Second

Second try

> Root

# Third

Third try

> Root
```
2. Verify that the text "Time to try" is directly above the go-to command.
3. Play the script.

**Expected Result**
"Time to try" is seen in the preview.  Or an error showing that this text will never be reached.

**Actual Result**
The script plays while quickly skipping over text it seems like it never existed.

**Desktop (please complete the following information):**
 - OS: Windows 
 - Browser: Chrome
 - Version: 99.0.4844.84

**Comment by lovelle-cardoso** [comment]
@Wipeless Fixed by d979812e 

(NOTE: If you don't leave a blank line after a text command, it will auto advance to the next command once the text finishes typing out)

**Comment by Wipeless** [comment]
Good to know!  This has been verified!
